enchant.lic

```diff
@@ -77,20 +78,35 @@ class Enchant
     clean_brazier
     bput("get my #{@item}", 'You get a')
     2.times do
-      bput("put my #{@item} on #{@brazier}", 'You glance down', 'With a flick')
+      case bput("put my #{@item} on #{@brazier}", 'You glance down', 'With a flick', 'You must first clean')
+      when 'You must first clean'
+        echo "Something is still on the brazier.  Cannot continue"
```

Comment:
❓ have you run into this case, where `clean_brazier` didn't clean it sufficiently?

Reply:
I just did in testing because the script wasn't completing correctly and leaving items on the brazier.  We'll need a better mechanism to deal with that edge case.  It's on my list as a future activity

Reply:
Sounds good